Lesson6/gokhan/styles.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 28px;
+  height: 60px;
+  box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, .05);
+}
+
+.nav-list {
+  display: flex;
+  gap: 10px;
+  font-size: 12px;
+}
+
+.btn {
+  min-width: 50px;
+  padding: 10px 15px;
+  border-radius: 4px;
+}
+
+.btn-outlined {
+  border: 1px solid #86B6F6;
+  color: #007BFF;
+}
+
+.btn-outlined:hover {
+  background-color: #86B6F6;
+  color: #fff;
+}
+
+.btn-contained {
+  color: #fff;
+  background-color: #007BFF;
+}
+
+.btn-contained:hover {
+  background-color: #fff;
+  color: #007BFF;
+}
+
+.hero-content {
+  text-align: center;
+  max-width:600px;
+  margin:48px auto 24px;
+}
+
+.cards-container{
+    display: flex;
+    gap: 24px;
+    justify-content: center;
+    flex-wrap: wrap;
+    background-color: #DEE2E6;
+}
+
+.card {
+  min-width: 235px;
+  border: 1px solid rgba(0, 0, 0, 0.125);
+  text-align: center;
+  border-radius: 4px;
+  
+  padding: 12px;
+  margin: 0;
+  display:flex;
+  flex-direction: column;
+  
+  border-top:1px solid rgba(0, 0, 0, 0.125);
+}
+
+.card > h2 {
+  background-color: #DEE2E6;
+  padding: 12px;
+  margin: 0;
+ 
+ }
+ .card-content{
+     padding:20px;
+     display:flex;
+     flex-direction: column;
+     gap: 24px;
+     border-top:1px solid rgba(0, 0, 0, 0.125);
+ }
+
+.price{
+    font-size:40px;
+    color:#000;
+}
+
+.interval{
+    font-size:32px;
+    color:#6c757d;
+}
+
+.card-content > ul {
+  color: #212529;
+}
+
+hr { 
+  margin: 4em auto;
+  max-width: 800px;
+  border-width: 0.5px;
+  color: #6c757d;
+} 
+
+.footers-container {
+  display:flex;
+  flex-direction: row;
+  justify-content: center;
+  max-width: 800px;
+}
+
+.footer > h4{
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    font-weight: 500;
+}
+
+.footer{
+  margin: 4rem auto;
+  max-width: 800px;
```

Added 'flex:1' to make all elements in the footer equal in width
```suggestion
  max-width: 800px;
  flex: 1;
```